test/functional/ui/screen.lua

```diff
@@ -392,7 +398,7 @@ function Screen:expect(expected, attr_ids, attr_ignore)
         .. ') differs from configured height(' .. #actual_rows .. ') of Screen.'
       end
       for i = 1, #actual_rows do
-        if expected_rows[i] ~= actual_rows[i] then
+        if expected_rows[i] ~= actual_rows[i] and expected_rows[i] ~= "{IGNORE}|" then
```

Comment:
@bfredl 	should this IGNORE feature be mentioned in the Screen:expect docstring?

Reply:
Seconded, I wasn't aware of it, and it will be pretty useful to me :+1:

Reply:
feel free to add it.

Reply:
Also some ext keys are missing, but that is not the same kind of deal: `print_snapshot()` will add them, so one doesn't need to know about them to use them.